modules/ztp-spoke-factory-pipeline.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="edge-factory-pipeline_{context}"]
+= The edge factory pipeline
+
+This stage deploys and configures the edge clusters. When this pipeline is completed, the edge clusters are ready for use when the enclosure gets relocated to the end customer's remote site.
+
+The flow associated with deploying the edge cluster is:
+
+Check hub::
+
+This step installs the various tools on the edge cluster that are needed. It downloads `jq`, `oc`, `opm` and `kubectl`. It proceeds to verify that various hub install prerequisites exist before proceeding, for example it checks the:
+
+* {product-title} version.
+* Nodes are ready.
+* Cluster Operators are ready.
+* Metal3 pods are ready.
+* Persistent volumes are created.
+* DNS requirements are satisfied.
+
+Deploy edge::
+
+This step starts with the edge cluster provisioning. This process ends with pushing a notification from the edge cluster to the hub and answering with an ACK.
+
+Deploy NMState and MetalLB::
+
+This step deploys NMState and the MetalLB operators that creates one profile per node to obtain an IP from the external network's DHCP. Then the MetalLB creates a resource called an AddressPool to build the relationship between the internal and external interface using a LoadBalancer interface. Finally it creates a service for the API and the ingress. Without this step you will not be able to access the API or ingress by using the external address.
```

Comment:
"operators that creates"  

Suggestion:
This step deploys NMState and the MetalLB operators.
NMState creates one profile per node to obtain an IP from the external network's DHCP